scripts/ironic/playbooks/ironic-pre-config.yml

```diff
@@ -0,0 +1,112 @@
+---
+# Copyright 2017, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+- name: Pre install config for ironic
+  hosts: localhost
+  tasks:
+    - name: Create ironic network configurations
+      blockinfile:
+        dest: /etc/network/interfaces.d/osa_ironic_interfaces.cfg
+        create: yes
+        block: |
```

Comment:
These blocks look fine however I'd like to see us have a little consistency in how we're arranging the config options. If we could follow the pattern here [ https://github.com/openstack/openstack-ansible/blob/master/etc/network/interfaces.d/aio_interfaces.cfg#L9-L16 ] I'd be appreciated.